main_app/templates/main_app/home.html

```diff
@@ -183,6 +190,14 @@ <h4 class="User"> &nbsp;&nbsp; {{user.username}} </h4>
     <div><img class="responsive" width="100%" src="{% static 'Images/rescues.gif' %}"></div>
   </div>
 
+  <div class="objectives" style="text-align: center;padding:60px;">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Akaya+Telivigala&display=swap" rel="stylesheet">
```

Comment:
Please change the font style.

Reply:
Ok @sakshikhachane , I will do the requested changes